package.json

```diff
@@ -21,6 +21,8 @@
   },
   "homepage": "https://github.com/brigade/eslint-config-brigade#readme",
   "dependencies": {
```

Comment:
Not sure I agree. They aren't dev dependencies, and they should get installed when you do `npm install`. The package will not function (stand-alone) without them.